src/DynamoCoreWpf/ViewModels/Menu/PreferencesViewModel.cs

```diff
@@ -782,9 +781,67 @@ public PreferencesViewModel(DynamoViewModel dynamoViewModel)
             var packageLoader = dynamoViewModel.Model.GetPackageManagerExtension()?.PackageLoader;            
             PackagePathsViewModel = new PackagePathViewModel(packageLoader, loadPackagesParams, customNodeManager);
 
+            PackagePathsViewModel.PackagePathOperationsEvent += PackagePathsViewModel_PackagePathOperations;
```

Comment:
I would say it would be good to clear this subscription if we can - it's always hard for me to guess which ones will cause memory leaks during testing.

Reply:
Thanks, I missed that.